src/components/Navbar/interfaces.tsx

```diff
@@ -1,12 +1,14 @@
 export interface Brand {
   /** Clinic/Hospital name */
   label: string
+  /** Label color */
+  color?: string
   /** Determina the href */
   href?: string
```

Comment:
I know that this PR did not include any changes to `href` prop. But it can be removed.